lib/const.rb

```diff
@@ -151,12 +172,17 @@
 when 'aarch64', 'armv7l'
   CREW_TGT = 'armv7l-cros-linux-gnueabihf'
   CREW_BUILD = 'armv7l-cros-linux-gnueabihf'
+  # These settings have been selected to match debian armhf.
+  # Using -mfpu=neon breaks builds such as webkit2gtk.
+  CREW_ARCH_FLAGS = '-mfloat-abi=hard -mtls-dialect=gnu -mthumb -mfpu=vfpv3-d16 -mlibarch=armv7-a+fp -march=armv7-a+fp'
```

Comment:
These flags are making my head spin.

Reply:
The `-mfpu=neon` code paths are untested on most software because the major distributions don't support it.